extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -178,6 +178,12 @@ export class AmpAnalytics extends AMP.BaseElement {
 
   /** @override */
   unlayoutCallback() {
+    if (Services.viewerForDoc(this.getAmpDoc()).isVisible()) {
+      // amp-analytics tag was just set to display:none. Page is still loaded.
+      return super.unlayoutCallback();
```

Comment:
Let's just `return false`. There's no telling what we make the base method do later on.

Reply:
Done.